### PR TITLE
Updated bot nuget

### PR DESCRIPTION
**Pull request by mamaso**

### CODE REVIEW

File: test/WebJobs.Script.Tests.Integration/App.config [hunk 1]
```diff
@@ -1,112 +1,112 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
```
Comment (mathewc) [inline test/WebJobs.Script.Tests.Integration/App.config]: looks like you're introducing whitespace changes somehow? Can these be reverted?

**Comment by fabiocav** [comment]
Do we have details on what changed?

**Comment by mamaso** [comment]
@fabiocav https://github.com/Microsoft/BotBuilder-WebJobs-BotExtension/pull/3

File: test/WebJobs.Script.Tests.Integration/App.config (hunk 1 above)
Comment (mamaso) [inline test/WebJobs.Script.Tests.Integration/App.config]: I can exclude them, but I'd like to see if you get the same behavior from VS on a nuget package update

Comment (mamaso) [inline test/WebJobs.Script.Tests.Integration/App.config]: App.config is currently checked in with LF line endings. Pretty sure VS is always going to complain about that, so we might as well flip it

**Pull request merged**